scene/gui/tree.cpp

```diff
@@ -2239,13 +2239,13 @@ void Tree::_gui_input(Ref<InputEvent> p_event) {
 			_go_left();
 		}
 
-	} else if (p_event->is_action("ui_up") && p_event->is_pressed()) {
+	} else if (p_event->is_action("ui_up") && p_event->is_pressed() && !k->get_control()) {
```

Comment:
Fix is fine, but use get_command instead of get_control, so it works for OSX users too.